src/Sql/dbo/Stored Procedures/User_ReadByEmails.sql

```diff
@@ -0,0 +1,19 @@
+﻿CREATE PROCEDURE [dbo].[User_ReadByEmails]
+    @Emails AS [dbo].[EmailArray] READONLY
+AS
+BEGIN
+    SET NOCOUNT ON;
+
+    IF (SELECT COUNT(1) FROM @Emails) < 1
+    BEGIN
+        RETURN(-1)
+    END
```

Comment:
❓ Is this necessary? Seems like it wouldn't happen via the code usage and even if it did it'd return an empty result right?

Reply:
It very well might not be necessary. However, the existing `User_ReadByIds` that I used as inspiration for this stored proc had the logic so I copied it to be safe. Do you want me to remove it?

<img width="335" alt="image" src="https://github.com/bitwarden/server/assets/116684653/5633acfb-c258-420f-a5cc-5cf6842c37a5">

Reply:
if there is not data to process it would be ideal to not make the database call

Reply:
Ok, I'll keep the early return. Thank you for the discussion though!